wlc/test_data/api/translations-hello-weblate-cs-file--POST--file=test-upload-data--method=translate

```diff
@@ -1,3 +1 @@
-{
-    "result": true
-}
+{"not_found":1,"skipped":141,"accepted":0,"total":143,"result":true,"count":143}
```

Comment:
The `result` would be false in this case, see server side code:

https://github.com/WeblateOrg/weblate/blob/ed5099d90fe545f09a327d1985ff30d907e1863a/weblate/api/views.py#L538-L539